jpasskit/src/main/java/de/brendamour/jpasskit/PKPass.java

```diff
@@ -268,10 +265,8 @@ public PKBarcode getBarcode() {
     @Deprecated
     public void setBarcode(final PKBarcode barcode) {
```

Comment:
Maybe we can drop the whole method as users of iOS 8 (and older versions) are quit uncommon as of now.

Reply:
Good idea!